static/js/cohort_details.js

```diff
@@ -327,13 +326,7 @@ require([
         return false;
     });
 
-    // If this is a new cohort, set TCGA Project selected as default
```

Comment:
No need for another request. If you push another change to this branch, the pull request gets updated.